.circleci/config.yml

```diff
@@ -157,6 +187,9 @@ jobs:
       - save-cache
 
 workflows:
+  detect-breaking-changes:
```

Comment:
Do we need to have a separate workflow for this task?

Reply:
need, no. Can, yes) Merged for simplicity.